lib/mapper.js

```diff
@@ -60,41 +60,40 @@ exports.page = function(page){
  * @api private
  */
 
-exports.track = function(track) {
+exports.track = function(track, settings) {
   // If the event name contains "unsubscribe", instead send an unsubscribe payload
   if (/[Uu]nsubscribe/.test(track.event())) {
     return reject({
-      auth_token: this.settings.authToken,
+      auth_token: settings.authToken,
       id: track.userId(),
       extras: {
         created_at: time(track.timestamp()),
         source: 'segment'
       }
     });
+  } 
+  var identity; 
+  if (typeof track.proxy('context.traits.email') !== 'undefined') {
+    identity = {
+      id: track.userId(),
+      email: track.proxy('context.traits.email')
```

Comment:
I think we should change this to use the facade function [track.email](https://github.com/segmentio/facade/blob/master/lib/track.js#L231)